graphene_sqlalchemy/types.py

```diff
@@ -76,20 +77,28 @@ class Meta:
         super(ORMField, self).__init__(_creation_counter=_creation_counter)
         # The is only useful for documentation and auto-completion
         common_kwargs = {
-            'model_attr': model_attr,
-            'type_': type_,
-            'required': required,
-            'description': description,
-            'deprecation_reason': deprecation_reason,
-            'batching': batching,
+            "model_attr": model_attr,
```

Comment:
Not to use double quotes, would be very nice in code. To comply with pep8 and flake8 linter.

Reply:
I agree, didn't comment much on the formatting since my review was about the implementation.
I think running `pre-commit run --all-files` should fix the quotes.

Reply:
@Fred7b I ran black over the code I touched, because my IDE automatically did so and it just switched to double quotes 😉 As mentioned in the description of the PR I can happily revert this, but as I couldn't find any restriction on the quoting in the docs except for the pre-install hook should pass, which it does, I wanted to go a similar route @erikwrede to solve the big road-blocks first and then adapt the style 👍

Reply:
@jendrikjoe actually, I decided to add black to pre-commit, so I have to revert my previous statement about the quotes.
I saw that graphene also uses black and I want to keep this codebase as similar to graphene as possible when it comes to tooling (#352)

Reply:
Nice 👍 🚀